confd/templates/functional-tests/kilda.properties.tmpl

```diff
@@ -8,8 +8,8 @@ elasticsearch.endpoint={{ getv "/kilda_tests_logging_elasticsearch_proto" "n/a"
 zookeeper.connect_string = {{ getv "/kilda_tests_zookeeper_hosts" "n/a" }}/{{ getv "/kilda_zookeeper_state_root" }}
 kafka.bootstrap.server={{ getv "/kilda_tests_kafka_hosts" "n/a" }}
 kafka.bootstrap.server.internal={{ getv "/kilda_tests_kafka_hosts_internal" "n/a" }}
-legacy.tsdb.endpoint=http://{{ getv "/kilda_tests_opentsdb_hosts" }}:{{ getv "/kilda_tests_opentsdb_port" }}
-tsdb.endpoint=http://{{ getv "/kilda_tests_victoriametrics_hosts" }}:{{ getv "/kilda_tests_victoriametrics_port" }}
+legacy.tsdb.endpoint=http://{{ getv "/kilda_opentsdb_hosts" }}:{{ getv "/kilda_opentsdb_port" }}
+tsdb.endpoint=http://{{ getv "/kilda_victoriametrics_host" }}:{{ getv "/kilda_victoriametrics_write_port" }}{{ getv "/kilda_tests_victoriametrics_query_path" }}
```

Comment:
Note: I think there is a typo here. here we have write port: `kilda_victoriametrics_write_port` but I think tests only read. Maybe we need read port?

Reply:
Good catch. Fixed.